networkit/cpp/graph/test/GraphToolsGTest.cpp

```diff
@@ -1047,4 +1040,43 @@ TEST_P(GraphToolsGTest, testEdgesSortedByWeight) {
         EXPECT_TRUE(G.checkConsistency());
     }
 }
+
+TEST_P(GraphToolsGTest, testEdgesRandomizer) {
+    if (!weighted())
+        return;
+    Aux::Random::setSeed(1, true);
+    ErdosRenyiGenerator generator(2000, 0.3, directed());
+    Graph G1 = generator.generate();
```

Comment:
You can merge both lines to `auto G1 = ErdosRenyiGenerator{2000, 0.3, directed()}.generate();`